onnxruntime_extensions/_hf_cvt.py

```diff
@@ -22,11 +22,22 @@ def __init__(self, tokenizer):
         self.tokenizer = tokenizer
 
     def bpe_tokenizer(self, **kwargs):
-        hf_gpt2_tokenizer = self.tokenizer
+        vocab_file = "./test/data/gpt2.vocab"
```

Comment:
that was the original idea but self.tokenizer in this case is usually a GPT2TokenizerFast object (or some other fast tokenizer) and does not have an encoder that I can get the vocab/merges from (previous imp was throwing an error - explained on Teams). I tried getting the path to the vocab or merges at least from init_kwargs but that also did not exist for the current test case, but I can add a check so it tries to get the vocab and merges from self.tokenizer.

Reply:
This doesn't work because of the vocab is not always same. If TokenizerFast doesn't have enough info, we use the slow version to extract the vocab and merges from it.

Reply:
Got it, I was thinking the same thing and realized this needed to be fixed. So one other problem is what string to pass into `from_pretrained`, for example in `exec(f"hf_bpe_tokenizer = {slow_tokenizer}.from_pretrained('gpt2')")`? I can extract the name of the slow tokenizer by just stripping "Fast" from the name like we do for cls_name) but the string for `from_pretrained` could be different. Will add an implementation now (like taking the string before "Tokenizer") but  it might not work for all cases, so will have to think of a better fix in the future.